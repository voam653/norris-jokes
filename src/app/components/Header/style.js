import styled from 'styled-components';

export const Container = styled.header`
    width: 100%;
`
export const Title = styled.h1``
