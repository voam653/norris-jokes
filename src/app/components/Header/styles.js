import styled from 'styled-components';

export const Container = styled.header`
    width: 100%;
    background-color: #250048;
    padding: 12px 24px;
    margin-bottom: 24px;
`

export const Title = styled.h1`
    color: white;
    font-size: 28px;
    margin: 0;
`
