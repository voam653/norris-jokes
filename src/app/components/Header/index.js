import React, { Component } from 'react';
import { Container, Title } from './styles';

class Header extends Component {
    render() {
        return (
            <Container>
                <Title> Jokes Categories </Title>
            </Container>
        )
    }
}

export default Header;