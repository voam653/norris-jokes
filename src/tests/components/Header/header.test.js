/**
 * @jest-environment jsdom
 */
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Header from '../../../app/components/Header';

describe('HeaderComponent', () => {
    const renderer = new ShallowRenderer();

    describe('Snapshots', () => {
      it('Should be a simple HeaderComponent', () => {
        const component = renderer.render(
          <Header />,
        );
        expect(component).toMatchSnapshot();
      });
    });
});