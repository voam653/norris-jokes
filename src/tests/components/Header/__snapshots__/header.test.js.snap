// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderComponent Snapshots Should be a simple HeaderComponent 1`] = `
<ForwardRef(styled.header)>
  <ForwardRef(styled.h1)>
     Jokes Categories 
  </ForwardRef(styled.h1)>
</ForwardRef(styled.header)>
`;
